include/bm/bluetooth/services/ble_dis.h

```diff
@@ -22,22 +22,47 @@
 #define BLE_DIS_H__
 
 #include <stdint.h>
+#include <bm/bluetooth/services/common.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/** @brief Default security configuration. */
+#define BLE_DIS_CONFIG_SEC_MODE_DEFAULT                                                            \
+	{                                                                                          \
+		.device_info_char.read = BLE_GAP_CONN_SEC_MODE_OPEN,                               \
+	}
+
 /**
- * @brief Function for initializing the Device Information Service.
+ * @brief Device Information service configuration.
+ */
+struct ble_dis_config {
+	/** Security configuration. */
+	struct {
+		/** Device information characteristic */
+		struct {
+			/** Security requirement for reading device information characteristic
+			 *  value.
+			 */
+			ble_gap_conn_sec_mode_t read;
+		} device_info_char;
```

Comment:
This sec mode read configuration value is used for all the characteristics in the dis service. That is not apparent when looking at this. Maye something like this could improve it.
```suggestion
			/** Security requirement for reading characteristic values of the device information service.
			 */
			ble_gap_conn_sec_mode_t read;
		} device_info_chars;
```